contracts/Vault.sol

```diff
@@ -183,14 +187,21 @@ contract Vault is
         external
         onlyRole(DEFAULT_ADMIN_ROLE)
     {
-        require(
-            PercentMath.validPerc(_perfFeePct),
-            "Vault: invalid performance fee"
-        );
+        require(_perfFeePct.validPerc(), "Vault: invalid performance fee");
         perfFeePct = _perfFeePct;
         emit PerfFeePctUpdated(_perfFeePct);
     }
 
+    function setInvestmentFeeEstimatePct(uint16 pct)
+        external
+        onlyRole(DEFAULT_ADMIN_ROLE)
```

Comment:
am confused between new SETTINGS_ROLE in other PR and DEFAULT_ADMIN_ROLE

do we need both?

Reply:
no, this should be SETTINGS_ROLE as well. but that only comes in on the next PR